internal/api/shared/migrate/mapping.cuepart

```diff
@@ -20,7 +20,7 @@ import (
     // mapping table for the calculation attribute (key = grafana unit, value = perses equivalent)
     calc: {
         first: "First"
-        firstNotNull: "First"
+        firstNotNull: "FirstNumber"
```

Comment:
Now that `FirstNumber` exists, migrate to that instead because it's more correct.